doc/custom-readers.md

```diff
@@ -78,13 +78,7 @@ ensuring backwards compatibility.
 
 # Bytestring readers
 
-Pandoc expects text input to be UTF-8 encoded. However, formats
-like docx, odt, epub, etc. are not text but binary formats. To
-read them, pandoc supports `ByteStringReader` functions. These
-functions work just like the `Reader` function that process text
-input, but instead of a list of sources, `ByteStringReader`
-functions are passed a bytestring, i.e., a string that contains
-the binary input.
+Pandoc expects text input to be UTF-8 encoded. In order to read binary formats, including docx, odt, and epub, pandoc supports ByteStringReader functions. These functions are similar to the Reader function that processes text input. Instead of a list of sources, ByteStringReader functions are passed a bytestring, i.e., a string that contains the binary input. 
```

Comment:
Thank you! There is one `Reader` or `ByteStringReader` function per file, would singular "function" make more sense?

Reply:
maybe, which version is more readable?

>In order to read binary formats, including docx, odt, and epub, pandoc supports (the?) `ByteStringReader` function. A `ByteStringReader` function is similar to the `Reader` function that processes text input. Instead of a list of sources, the `ByteStringReader` function is passed a bytestring, i.e., a string that contains the binary input.

Reply:
Sorry, this got lost. I like the above :+1:

Reply:
I didn't want this to get lost.  I think the above looks good (maybe with `the`?).
If @IlonaSilverwood doesn't want to update it, maybe @tarleb or I could do a quick online edit; it also needs to be wrapped within 80 columns.

Reply:
Ilona and I talked briefly, an updated version is incoming.